### PR TITLE
Add server configuration and feature toggle pipe

### PR DESCRIPTION
**Pull request by MaxKless**
## PR Type

[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no API changes)
[ ] Build-related changes
[ ] CI-related changes
[ ] Documentation content changes
[ ] Application / infrastructure changes
[ ] Other: 

## What Is the current and new behavior?

-  Currently, there is no way to check server settings from a template.  This PR adds a ishServerSetting pipe that checks the store for relevant server settings.

- This PR adds an alternative to the feature toggle directive by introducing a feature toggle pipe. 

## Does this PR Introduce a Breaking Change?

[ ] Yes
[x] No

### CODE REVIEW

File: docs/concepts/configuration.md [hunk 1]
```diff
@@ -186,6 +186,12 @@ If the feature is deactivated, the user is sent to the error page on accessing.
 <ish-product-add-to-compare *ishFeature="'compare'"> ...</ish-product-add-to-compare>
 ```
 
+**Pipe**
+
+```html
+<ish-product-add-to-compare *ngIf="'compare' | ishFeature"> ...</ish-product-add-to-compare>
```
Comment (SGrueber) [inline docs/concepts/configuration.md:192]: What's the reason to have a pipe and a directive? Is there a difference between them?

File: src/app/core/facades/app.facade.ts [hunk 2]
```diff
@@ -90,6 +87,10 @@ export class AppFacade {
     return isAppTypeREST && !isBusinessCustomer ? 'privatecustomers' : 'customers';
   }
 
+  serverSetting$<T>(path: string) {
```
Comment (SGrueber) [inline src/app/core/facades/app.facade.ts:90]: for me it is not clear, what the path means, which is the starting point of the path

File: src/app/core/pipes/feature-toggle.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { FeatureToggleService } from 'ish-core/utils/feature-toggle/feature-toggle.service';
+
+@Pipe({ name: 'ishFeature', pure: true })
```
Comment (SGrueber) [inline src/app/core/pipes/feature-toggle.pipe.ts:5]: docu, see feature toggle directive

File: src/app/core/pipes/server-setting.pipe.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { OnDestroy, Pipe, PipeTransform } from '@angular/core';
+import { Subject, Subscription } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+import { AppFacade } from 'ish-core/facades/app.facade';
+
+@Pipe({ name: 'ishServerSetting', pure: false })
```
Comment (SGrueber) [inline src/app/core/pipes/server-setting.pipe.ts:7]: docu

File: docs/concepts/configuration.md [hunk 5]
```diff
@@ -186,6 +186,12 @@ If the feature is deactivated, the user is sent to the error page on accessing.
...
+```
+
```
Comment (SGrueber) [inline docs/concepts/configuration.md:194]: There should be at least 1 code example for each of these 2 pipes

**Review by SGrueber — CHANGES_REQUESTED** [review]

File: docs/concepts/configuration.md (hunk 1 above)
Comment (MaxKless) [reply docs/concepts/configuration.md:192]: they are functionally the same but we decided to implement an alternative for cases where you'd need more than one directive which isn't possible on a single component.

Comment (shauke) [reply docs/concepts/configuration.md:192]: 👍

**Review by shauke — APPROVED** [review]
Looks good

**Review by SGrueber — APPROVED** [review]

**Pull request merged**